crates/pipeline_manager/src/db/test.rs

```diff
@@ -3,10 +3,11 @@ use super::{
     storage::Storage, AttachedConnector, ConfigDescr, ConfigId, ConnectorDescr, ConnectorId,
     ConnectorType, PipelineId, ProjectDB, ProjectDescr, ProjectId, ProjectStatus, Version,
 };
-use crate::db::DBError;
+use crate::db::{pg_setup, DBError};
```

Comment:
so `mod test` in mod.rs maybe also needs to be conditionally included with cfg(feature = dev)?

Reply:
I've updated the CI workflows to have feature=dev wherever --no-default-features is called.

Reply:
I believe it will result in a compilation error if you do `cargo test --no-default-features` which is a bit awkward. You could update to include the `crate::db::pg_setup` cfg(feature = "dev") and in addition also for `cfg(test)`?

Reply:
e.g., `cfg(any(test, feature = "dev")` might do the trick?

Reply:
Do you mean `all(test, feature = "dev)` ?